src/routes/+page.svelte

```diff
@@ -16,7 +16,7 @@
     <title>SnowballR</title>
 </svelte:head>
 <SimpleNavigationBar loadingTitle={Promise.resolve("SnowballR")} {user} />
-<main class="mb-10 flex h-full w-full flex-row gap-x-5 overflow-hidden">
+<main class="mb-10 flex h-full w-full flex-row gap-x-5 overflow-hidden px-5">
```

Comment:
```suggestion
<main class="mb-10 flex h-full w-full flex-row gap-x-15 overflow-hidden px-5">
```
I agree that it is smarter to handle the padding on the homepage and not within the `NamedList` component, however the gap between the two list on the homepage very small

Reply:
This is a good idea, but in my opinion 15 is too much. I think 10 would be a better fit.

```suggestion
<main class="mb-10 flex h-full w-full flex-row gap-x-10 overflow-hidden px-5">
```

Reply:
Yes, this is a good idea! But I think 15 is a bit much. Please take a look at the new changes, there i used 10 instead. I also added another gap in another file which we missed.

Reply:
I agree, 10 is better.